generators/2.0/java/index.js

```diff
@@ -1,6 +1,7 @@
 const Generator = require('yeoman-generator');
 const colors = require('colors');
 const path = require('path');
+const fs = require('fs');
```

Comment:
No need to import `fs` if I remember well, you can directly use `this.fs` in generator class

Reply:
the `Generator.fs` doesn't contain all the methods which are exposed by the `fs` from NodeJs

Reply:
I need both of them. 
The generator `fs` is developed here: https://github.com/SBoudrias/mem-fs-editor and it's different from the NodeJS `fs`

Reply:
I don't see where you directly use `fs` in this PR